### ISSUE TITLE
Error when loading imdb_reviews dataset

### ISSUE THREAD

**Issue by keosu** [tensorflow/datasets#2021]
**Short description**
the following code report an error when running on colab
```
train_data, validation_data, test_data = tfds.load(
    name="imdb_reviews", 
    split=('train[:60%]', 'train[60%:]', 'test'),
    as_supervised=True)
```

**Environment information**
colab

**Reproduction instructions**

1. open this link https://www.tensorflow.org/tutorials/keras/text_classification_with_hub
2. run in colab 

**Link to logs**
TypeError                                 Traceback (most recent call last)
<ipython-input-2-4b45eebfd65c> in <module>()
      4     name="imdb_reviews",
      5     split=('train[:60%]', 'train[60%:]', 'test'),
----> 6     as_supervised=True)

5 frames
/usr/local/lib/python3.6/dist-packages/tensorflow_datasets/core/download/download_manager.py in <genexpr>(.0)
    255   def downloaded_size(self):
    256     """Returns the total size of downloaded files."""
--> 257     return sum(size for size, sha256 in self._recorded_url_infos.values())
    258 
    259   def _get_final_dl_path(self, url, sha256):

TypeError: 'UrlInfo' object is not iterable

**Comment by vijayphoenix** [comment]
@keosu Thanks for reporting the issue.
This should be fixed soon by #2022

**Comment by Conchylicultor** [comment]
Sorry about the regression. The fix should appear in the next tf-nightly

**Comment by vijayphoenix** [comment]
Fixed by f096c13.
Can be closed
